### PR TITLE
Report configuration

### PR DESCRIPTION
**Pull request by Yuri05**
Report configuration for the "minimal" example.
Not all plots from the minimal example were added - just 1 plot with some curves and observed data.

### CODE REVIEW

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 1]
```diff
@@ -1,39 +1,99 @@
-{"MasterQualificationPlan": {
-    "Projects": [
+{
+"ReportConfigurationPlan": {
+    "SimulationMappings": [
         {
```
Comment (Yuri05) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: This is the new node with the mappings `{Project, Simulation}` :arrow_right: `Folder with simulation data`

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 2]
```diff
@@ -1,39 +1,99 @@
...
+        {
+            "refProject": "Itraconazole",
+            "refSimulation": "S1",
+            "path": "../Itraconazole/S1"
         }
     ],
-    "ObservedDataSets": [{
+    "ObservedDataSets": [
+	{
         "id": "PK_Ratio_CYP3A4",
```
Comment (Yuri05) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: "original" data set from the qualification plan

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 3]
```diff
@@ -1,39 +1,99 @@
...
-        "path": "../PK_Ratio_CYP3A4.xlsx",
+        "path": "../ObservedData/PK_Ratio_CYP3A4.xlsx",
         "type": "DDIRatio"
-    }],
+    },
+	{
+        "id": "Midazolam 600mg SD",
+        "path": "../ObservedData/Midazolam 600mg SD.xlsx",
```
Comment (Yuri05) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: "INTERNAL" data set exported from PK-Sim project

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 4]
```diff
@@ -1,39 +1,99 @@
...
+    },
+	{
+        "id": "Intraconazole 600mg MD",
+        "path": "../ObservedData/Intraconazole 600mg MD.xlsx",
+        "type": "TimeProfile"
+    }
+	],
     "ReportConfiguration": {
         "Plots": {
-            "AllPlots": [
```
Comment (Yuri05) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: AllPlots are replaced with concrete plot definitions
(in the 1st iteration just one plot as example)

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 5]
```diff
@@ -1,39 +1,99 @@
...
-                    "refSimulation": "S1",
-                    "location": 2
-                },
-                {
-                    "refProject": "Midazolam",
-                    "refSimulation": "S2",
-                    "location": 3
-                }
-            ],
+			"TimeProfile":
```
Comment (Yuri05) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: one time profile plot with 2 simulated curves and 1 observed data set

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 6]
```diff
@@ -1,39 +1,99 @@
...
+			{
+				"name": "Another Chart",
+				"description": "", 
+				"refProject": "Midazolam",
+				"refSimulation": "S1",
+				"legendPosition": "RightInside",
+				"location": 2,
```
Comment (Yuri05) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: I think "location" is not good (too general) - we should replace it with "sectionId" or something like that

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 7]
```diff
@@ -1,39 +1,99 @@
...
+				"Axes" : [
+					{
```
Comment (Yuri05) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: Axes definitions: 1 X-axis and >=1 Y-axes

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 8]
```diff
@@ -1,39 +1,99 @@
...
+						"unit": "�mol/l",
+						"scale": "Log",
+						"minValue": null,
+						"maxValue": null
+					}					
+				],
+				"Curves": [
+					{
+						"axis": "Y",
+						"type": "Simulated",
```
Comment (Yuri05) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: type of the output: `Simulated `or `Observed`.
Depending on the type, `source `defines either the path of a simulated curve (e.g. `Brain/Plasma/Aspirin`) or the id of an observed data set (e.g. `Midazolam 600mg SD`)

File: examples/minimal/reporting engine input/report-configuration-plan.json [hunk 9]
```diff
@@ -1,39 +1,99 @@
...
+    },
+	{
+        "id": "Midazolam 600mg SD",
+        "path": "../ObservedData/Midazolam 600mg SD.xlsx",
+        "type": "TimeProfile"
+    },
+	{
+        "id": "Intraconazole 600mg MD",
+        "path": "../ObservedData/Intraconazole 600mg MD.xlsx",
+        "type": "TimeProfile"
```
Comment (msevestre) [inline examples/minimal/reporting engine input/report-configuration-plan.json]: TimeProfile? What does that mean for observed data?

File: examples/minimal/reporting engine input/report-configuration-plan.json (hunk 6 above)
Comment (msevestre) [reply examples/minimal/reporting engine input/report-configuration-plan.json]: sure

**Comment by msevestre** [comment]
@Yuri05 Assuming I should merge for now?

File: examples/minimal/reporting engine input/report-configuration-plan.json (hunk 9 above)
Comment (Yuri05) [reply examples/minimal/reporting engine input/report-configuration-plan.json]: Time series: time array, values array.
Maybe no type required here at all

**Pull request merged**